website/content/en/docs/dev/Internals/_index.md

```diff
@@ -71,7 +71,11 @@ VNC:
 - `vncpassword`: VNC display password
 
 Guest agent:
-- `ga.sock`: Forwarded to `/run/lima-guestagent.sock` in the guest, via SSH
+
+Each drivers use their own mode of communication 
+- `qemu`: uses virtio-port `io.lima-vm.guest_agent.0`
+- `vz`: uses vsock port 2222
+- `wsl2`: uses free random vsock port
```

Comment:
Can't we use a random port for vz too?

Reply:
Vz create port on per vm basis so there is no way to get it ahead of time. 

Since we are bootstrapping the vm i think we can be sure that it will be free